example_sns_init.yaml

```diff
@@ -25,96 +31,338 @@ description: >
+#      equal to the sum of all developer tokens.
 Distribution:
-
+    # The neurons distributed at genesis to the development team of the dapp
+    # being decentralized. Each element in this list specifies one such neuron,
+    # including its stake, controlling principal, unique memo identifying this
+    # neuron, dissolve delay, and a vesting period. Even though these neurons
+    # are distributed at genesis, they are locked in a (restricted)
+    # pre-initialization mode until the decentralization swap is completed.
     Neurons:
-
-        - principal: YOUR_PRINCIPAL_ID  # propose_sns.sh can fill this in automatically.
+        - principal: YOUR_PRINCIPAL_ID  # For testing, propose_sns.sh will fill this in automatically.
           stake: 1_000 tokens
-          memo: 0 # This is shown for illustrative purposes, and is not necessary.
+          memo: 0
           dissolve_delay: 2 years
           vesting_period: 4 years
-
-        # More neurons can be listed here. Usually, you would have
-        # separate neurons for each person on your team.
-
+    # The initial SNS token balances of the various canisters of the SNS.
     InitialBalances:
-        governance: 2_000_000 tokens # aka "the treasury"
-        swap:         500_000 tokens # See also the Swap section below.
+        # The initial SNS token balance of the SNS Governance canister is known
+        # as the treasury. This is initialized in a special sub-account, as the
+        # main account of Governance is the minting account of the SNS Ledger.
+        governance: 2_000_000 tokens
+
+        # The initial SNS token balance of the Swap canister is what will be
+        # available for the decentralization swap. These tokens will be swapped
+        # for ICP.
+        swap: 500_000 tokens
 
-    # Optional, but highly recommended. This is a literal
-    # checksum.
+    # Checksum of the total number of tokens distributed in this section.
     #          1_000    (neuron)
     #      2 million    (governance)
     # + 500 thousand    (swap)
     # --------------
     total: 2_501_000 tokens
 
+# Configuration of the decentralization swap parameters. Adjusting these fields
+# will increase or decrease the decentralization factor achieved in the
+# decentralization swap. Choose these parameters carefully, if a
+# decentralization swap fails, the SNS will restore the dapp canister(s) to
+# the fallback controllers (defined in `fallback_controller_principals`) and you
+# will need to start over.
 Swap:
+    # The minimum number of participants that must participate for the
+    # decentralization swap to succeed. This guarantees that a newly created SNS
+    # will have a minimum number of unique users (decentralization factor).
+    # If a decentralization swap's maximum target is reached, or the
+    # deadline elapses and there have not been at least `minimum_participants`,
+    # the swap will fail.
```

Comment:
```suggestion
    # The minimum number of participants that must participate for the
    # decentralization swap to succeed. 
    # If a decentralization swap finishes, either because the maximum target is reached or the
    # deadline elapses, and if there have not been at least `minimum_participants`,
    # the swap will fail.
```